### PR TITLE
AG-10700: Add videos to docs

### PR DESCRIPTION
**Pull request by JamesSwinton**
https://ag-grid.atlassian.net/browse/AG-10700

Add VideoSection component to docs and add videos to Create a Basic Chart doc

### CODE REVIEW

File: packages/ag-charts-website/src/design-system/modules/VideoSection.module.scss [hunk 1]
```diff
@@ -12,8 +12,8 @@
     }
 }
 
-.header {
-    font-size: var(--text-fs-lg);
+.header > * {
```
Comment (markdurrant) [inline packages/ag-charts-website/src/design-system/modules/VideoSection.module.scss]: You can refactor this selector to get rid of the `!important`. 

```
.videoSection .header > * {
    font-size: var(--text-fs-xl);
}
```

**Review by markdurrant — COMMENTED** [review]
Generally all looking good 👍 

Left a comment on one CSS niggle.

File: packages/ag-charts-website/src/design-system/modules/VideoSection.module.scss (hunk 1 above)
Comment (JamesSwinton) [reply packages/ag-charts-website/src/design-system/modules/VideoSection.module.scss]: Thanks, added this in

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**